web/tests/peer_data.test.cjs

```diff
@@ -282,27 +282,42 @@ test("maybe_fetch_stream_subscribers", async () => {
     assert.deepEqual([...subscribers_after_fetch.keys()], [1, 2, 4, 7, 9]);
 
     peer_data.clear_for_testing();
-    assert.equal(await peer_data.maybe_fetch_is_user_subscribed(india.stream_id, 2), true);
+    assert.equal(await peer_data.maybe_fetch_is_user_subscribed(india.stream_id, 2, false), true);
     assert.equal(peer_data.has_full_subscriber_data(india.stream_id), true);
 
     peer_data.clear_for_testing();
     assert.equal(peer_data.has_full_subscriber_data(india.stream_id), false);
-    assert.equal(await peer_data.maybe_fetch_is_user_subscribed(india.stream_id, 2), true);
-    assert.equal(await peer_data.maybe_fetch_is_user_subscribed(india.stream_id, 5), false);
+    assert.equal(await peer_data.maybe_fetch_is_user_subscribed(india.stream_id, 2, false), true);
+    assert.equal(await peer_data.maybe_fetch_is_user_subscribed(india.stream_id, 5, false), false);
 
     channel.get = () => {
         throw new Error("error");
     };
     peer_data.clear_for_testing();
     blueslip.expect("error", "Failure fetching channel subscribers");
-    assert.equal(await peer_data.maybe_fetch_is_user_subscribed(india.stream_id, 5), null);
+    assert.equal(await peer_data.maybe_fetch_is_user_subscribed(india.stream_id, 5, false), null);
     // If we know they're subscribed, we return `true` even though we don't have complete
     // data.
     peer_data.bulk_add_subscribers({
         stream_ids: [india.stream_id],
         user_ids: [5],
     });
-    assert.equal(await peer_data.maybe_fetch_is_user_subscribed(india.stream_id, 5), true);
+    assert.equal(await peer_data.maybe_fetch_is_user_subscribed(india.stream_id, 5, false), true);
+
+    let num_attempts = 0;
+    channel.get = async () => {
+        num_attempts += 1;
+        if (num_attempts === 2) {
+            return {
+                subscribers: [1, 2, 3, 4],
+            };
+        }
+        throw new Error("error");
+    };
+    peer_data.clear_for_testing();
+    const subscribers = await peer_data.maybe_fetch_stream_subscribers(india.stream_id, true);
```

Comment:
If I remove this line (and the asserts below it) the issue where many many files lose code coverage went away, and I've been having trouble figuring out why.